### PR TITLE
Add Version instance

### PR DESCRIPTION
**Pull request by parsonsmatt**
This PR adds an instance for `base`'s `Data.Version.Version`

### CODE REVIEW

File: src/Data/Swagger/Internal/ParamSchema.hs [hunk 1]
```diff
@@ -218,6 +219,11 @@ instance ToParamSchema T.Text where
 instance ToParamSchema TL.Text where
   toParamSchema _ = toParamSchema (Proxy :: Proxy String)
 
+instance ToParamSchema Version where
+  toParamSchema _ = mempty
+    & type_ .~ SwaggerString
+    & pattern ?~ "^[\\d+.]*[\\-[\\w]+]*$"
```
Comment (phadej) [inline src/Data/Swagger/Internal/ParamSchema.hs]: Can we pretend there aren't *tags*? (E.g. `Cabal`'s version doesn't have them). Also e.g. `QuickCheck` pretends here aren't tags (http://hackage.haskell.org/package/QuickCheck-2.11.3/docs/src/Test-QuickCheck-Arbitrary.html#line-950) too.

Also the digit part is 

```
\d+(\.\d+)*
```

I.e.

- `0.1.` isn't valid version
- and dot `.` have to be escaped in regexp.

Comment (parsonsmatt) [reply src/Data/Swagger/Internal/ParamSchema.hs]: I'd rather not have a pattern than one that fails on valid values, personally.

Comment (parsonsmatt) [reply src/Data/Swagger/Internal/ParamSchema.hs]: I dropped the tags part of the schema and fixed it to be what you said :) thanks!

File: src/Data/Swagger/Internal/ParamSchema.hs [hunk 2]
```diff
@@ -218,6 +219,11 @@ instance ToParamSchema T.Text where
 instance ToParamSchema TL.Text where
   toParamSchema _ = toParamSchema (Proxy :: Proxy String)
 
+instance ToParamSchema Version where
+  toParamSchema _ = mempty
+    & type_ .~ SwaggerString
+    & pattern ?~ "^\\d+[\\.\\d+]*$"
```
[the remark this answers is not in the document]
Comment (parsonsmatt) [reply src/Data/Swagger/Internal/ParamSchema.hs]: yes, thanks for catching that :) my regex test suite didn't catch that one.

**Pull request merged**